### ISSUE TITLE
options for init command should have required arguments.

### ISSUE THREAD

**Issue by jamesgeorge007** [react-native-community/cli#1323]
## Description

Not supplying a value for any of the options available as part of the `init` command fails.

## Reproducible Demo

`react-native-cli init app --template`

<img width="552" alt="Screenshot 2020-11-23 at 2 15 07 PM" src="https://user-images.githubusercontent.com/25279263/99942884-618e6b80-2d96-11eb-8e43-58ca3060407f.png">

## Proposed Solution

```bash
error: option `--template <string>' argument missing
```

```diff
     {
-      name: '--version [string]',
+      name: '--version <string>',
       description: 'Shortcut for `--template react-native@version`',
     },
     {
-      name: '--template [string]',
+      name: '--template <string>',
       description:
         'Uses a custom template. Valid arguments are: npm package, absolute directory prefixed with `file://`, Git repository or a tarball',
     },
@@ -21,11 +21,11 @@ export default {
       description: 'Forces using npm for initialization',
     },
     {
-      name: '--directory [string]',
+      name: '--directory <string>',
       description: 'Uses a custom directory instead of `<projectName>`.',
     },
     {
-      name: '--title [string]',
+      name: '--title <string>',
       description: 'Uses a custom app title name for application',
      }
```

/cc @thymikee

**Comment by thymikee** [comment]
Yup, makes sense

**Comment by jamesgeorge007** [comment]
Fixed in #1328
